src/teams/team_profile.py

```diff
@@ -0,0 +1,160 @@
+from src.flaskapp.db import coll
+from src.flaskapp.util import aggregate_team_meta
+
+
+def get_team_profile(email, team_id):  # GET
+    """Get team profile
+
+    Fetches from the teams collection by using the team name as key.
+
+    Args:
+        User's email (str)
+        Team name (str)
+
+    Returns:
+        User profile object (dict)
+    """
+    team = coll("teams").find_one({"_id": team_id})
+    if not team:
+        return {"message": "Team does not exist"}, 400
+
+    if email not in team["members"]:
+        return {"message": f'User not in team "{team_id}"'}, 403
+
+    del team["meta"]
```

Comment:
I think we can specify to mongodb to not return the meta part. 
![image](https://user-images.githubusercontent.com/19612528/94601017-d6df3380-0260-11eb-9b78-0a75ca5ce85b.png)
Search for projection operators for Mongodb. I am also new to this as well.

Reply:
and likewise specify only things we want. This could drastically improve read speed. If stuff gets large. Ik database calls can get slow but idk if we need to be that strict with the scale of our app.

Reply:
tracking separately with https://github.com/HackRU/teamRU/issues/18